Documentation/docs-mobile/building-apps/build-items.md

```diff
@@ -194,7 +194,7 @@ The result of the above code snippet has a different effect for each
 * Java binding projects:
   * `foo.jar` maps to [**EmbeddedJar**](#embeddedjar).
   * `foo.jar` maps to [**EmbeddedReferenceJar**](#embeddedreferencejar)
-    if `Bind="false"` metadata is added.
+    if `Bind="false"` metadata (attribute) is added.
```

Comment:
I don't think that adding `(attribute)` is useful or clarifying: "metadata" is the MSBuild terminology for XML attributes on items, as per [Item element (MSBuild)](https://learn.microsoft.com/en-us/visualstudio/msbuild/item-element-msbuild?view=vs-2022):

> ## Specify metadata as attributes
> In MSBuild 15.1 or later, any metadata with a name that doesn't conflict with the current list of attributes can optionally be expressed as an attribute.

Additionally, attributes don't need to be used; nested elements can also be used:

```xml
<ItemGroup>
  <AndroidLibrary Update="example.jar">
    <Bind>false</Bind>
  </AndroidLibrary>
</ItemGroup>
```

*Nobody would do this*, but it's still supported.

"Metadata" is the terminology, and "metadata" is already used.

Reply:
@jonathanpeppers: re-reading things, though, I'm not immediately sure what it means in context:

> [For] Java binding projects:
> * `foo.jar` maps to EmbeddedJar.
> * `foo.jar` maps to EmbeddedReferenceJar if `Bind="false"` metadata is added.

Should it be `is added`, or should it be `is present`?

Additionally, with .NET 6 *there are no "Java binding projects" anymore*: both application projects and class lib projects can include Java libraries.

This feels like it could use a minor rewrite, and I'm not currently sure what that would look like.

Reply:
I guess it depends on what effect you want to have here. As an outsider, I can tell you that as written, it's difficult to understand what you need to do here, for the two situations that I think are most likely what people want to know:

- How to specify an AAR/JAR that you _want_ to bind
- How to specify an AAR/JAR to be included that you _don't want_ to bind

Those, I think, are important to have in clear examples.

Reply:
I think this is something that should be covered on a page at:
* https://learn.microsoft.com/dotnet/android